spec/requests/admin_payment_confirmation_report_upload_spec.rb

```diff
@@ -40,6 +40,12 @@
 
           expect(payroll_run.reload.confirmation_report_uploaded_by).to eq(admin_session_id)
         end
+
+        it "queues emails" do
```

Comment:
I would be tempted to add this expectation to the existing `it` above and reword the description to mention sending payment confirmation emails. Whilst separating out `it` examples to better explain behaviour is good in general, for expensive feature and request specs, each `it` we add increases the test suite time by a non-trivial amount, which adds up over time. Where we can avoid it, we should try, to help keep the test suite fast.